okhttp/src/main/java/okhttp3/internal/ws/MessageInflater.kt

```diff
@@ -0,0 +1,36 @@
+package okhttp3.internal.ws
+
+import okio.Buffer
+import okio.InflaterSource
+import java.io.Closeable
+import java.io.EOFException
+import java.io.IOException
+import java.util.zip.Inflater
+
+class MessageInflater : Closeable {
+  private val source = Buffer()
+  private val inflater = Inflater(true /* omit zlib header */)
+  private val inflaterSource = InflaterSource(source, inflater)
+
+  private var totalBytesToInflate = 0L
```

Comment:
should this be an Int ?

Reply:
(if we inflate more than 2 GiB of data, I assume totalIn wraps around?)

Reply:
Checked the javadocs for `totalIn` again:

```
     * Since the number of bytes may be greater than
     * Integer.MAX_VALUE, the {@link #getBytesRead()} method is now
     * the preferred means of obtaining this information.</p>
```

So, migrated to `getBytesRead` instead of moving this to int.